administrator/language/de-DE/com_fields.ini

```
@@ -9,10 +9,8 @@ COM_FIELDS_BATCH_GROUP_LABEL="Zum Verschieben oder Kopieren der Auswahl eine Gru
 COM_FIELDS_BATCH_GROUP_OPTION_NONE="- Keine Gruppe -"
 COM_FIELDS_CHOOSE_CONTEXT_LABEL="Kontext"
 COM_FIELDS_ERROR_UNIQUE_NAME="Ein anderes Feld hat denselben Namen (ggf. auch im Papierkorb oder in einer anderen Komponente)."
-COM_FIELDS_FIELDS_FILTER_SEARCH_DESC="Suche in Feld Name, Titel oder Notiz. ID: verwenden um eine Feld-ID zu suchen. AUTHOR: verwenden um nach Feld-Autor zu suchen."
-COM_FIELDS_FIELDS_TABLE_CAPTION="Tabelle der Felder"
-COM_FIELDS_FIELD_CLASS_DESC="Das CSS-Klassenattribut für das Feld in der Eingabemaske. Bei mehreren Klassen mit Leerzeichen trennen."
-COM_FIELDS_FIELD_CLASS_LABEL="CSS-Klasse Feld"
+COM_FIELDS_FIELD_CLASS_DESC="The class attributes of the field in the edit form. If multiple classes are needed, list them with spaces."
```

```suggestion
COM_FIELDS_FIELD_CLASS_DESC="Die Klassenattribute des Feldes im Bearbeitungsformular. Mehrere Klassen mit Leerzeichen getrennt aufführen."
```

Bearbeitungsformular ist noch unstimmig da bisher nie genutzt.